src/app/views/data/data.js

```diff
@@ -20,6 +20,9 @@ class DataViewController extends ADBMapController {
     this.$log.debug('soum code:', this.soumCode);
 
     this.soumData.geojson(this.soumCode).then(geojson => this._addGeoJSONLayer(geojson));
+    this.soumData.load(this.soumCode).then(soum => {
```

Comment:
Either way I don't feel strongly about it for this project, but worth thinking about in other larger projects down the line.